src/layout/OrgLayout.tsx

```diff
@@ -1,19 +1,27 @@
-import getOrg from '../fetching/getOrg';
-import { useQuery } from 'react-query';
+import DefaultLayout from './DefaultLayout';
+import OrgHeader from './OrgHeader';
 
 interface OrgLayoutProps {
     children: JSX.Element;
     orgId: string;
 }
 
 const OrgLayout = ({ children, orgId } : OrgLayoutProps) : JSX.Element => {
-    const orgQuery = useQuery(['org', orgId], getOrg(orgId));
```

Comment:
I would recommend against moving the `useQuery()` deeper and deeper into the component hierarchy. Instead, retrieve the organization data as far up as possible (but still adjacent to where it's needed) and pass the object down in a property instead.

This makes the components more "stupid", which makes them easier to test and less error-prone.